config/deploy.rb

```diff
@@ -12,3 +12,30 @@
 
 append :linked_files, '.env.production', 'public/robots.txt'
 append :linked_dirs, 'vendor/bundle', 'node_modules', 'public/system'
+
+namespace :systemd do
+  %i[sidekiq streaming web].each do |service|
+    desc "Reload #{service} service"
+    task "#{service}:reload".to_sym do
+      on roles(:app) do
+        systemctl :reload, service
+      end
+    end
+
+    desc "Show the status of #{service} service"
+    task "#{service}:status".to_sym do
+      on roles(:app) do
+        systemctl :status, service
+      end
+    end
+  end
+
+  def systemctl(action, service)
+    # runs e.g. "sudo restart mastodon-sidekiq.service"
+    sudo :systemctl, action, "#{fetch(:application)}-#{service}.service"
+  end
+end
+
+after 'deploy:publishing', 'systemd:web:reload'
+after 'deploy:publishing', 'systemd:sidekiq:reload'
```

Comment:
Basically, only `web` has a `reload`. The others need a `restart`.

Reply:
Understood. 

I've changed and refactored the code.

Rebased and force pushed too.